README.md

```diff
@@ -58,6 +58,10 @@ We welcome your contributions! Please see the [contribution guidelines](CONTRIBU
 
 For SDK feedback or to report a bug, please file a [GitHub Issue](https://github.com/Microsoft/Azure-Kinect-Sensor-SDK/issues). For general suggestions or ideas, visit our [feedback forum](https://aka.ms/azurekinectfeedback).
 
+## Join Our Developer Program
+
+Complete your developer profile [here](https://aka.ms/iwantmr) to get connected with our Mixed Reality Developer Program. You will receive the latest on our developer tools, events, and early access offers. 
```

Comment:
Using "here" as the text for a hyperlink is discouraged in a lot of HTML style guides. 

```suggestion
[Complete your developer profile](https://aka.ms/iwantmr) to get connected with our Mixed Reality Developer Program. You will receive the latest on our developer tools, events, and early access offers. 
```